packages/hop-node/src/watchers/BondWithdrawalWatcher.ts

```diff
@@ -185,6 +185,17 @@ class BondWithdrawalWatcher extends BaseWatcher {
         throw new BonderFeeTooLowError(msg)
       }
 
+      const hToken = await sourceL2Bridge.hToken()
+      const sourceChainTotalHTokens = await hToken.getTotalSupply()
+      const pendingAmount = await sourceL2Bridge.getPendingAmountForChainId(destinationChainId!)
+      const hTokensAvailable = sourceChainTotalHTokens.add(pendingAmount)
+      if (amount!.gt(hTokensAvailable)) {
```

Comment:
same comment as above -- if `amount` is defined here, our types could be improved. again, this is non-blocking